src/systems/systems/src/shared/arinc429.rs

```diff
@@ -40,6 +40,35 @@ impl<T: Copy> Arinc429Word<T> {
         self.ssm == SignStatus::NormalOperation
     }
 }
+impl Arinc429Word<f32> {
+    pub fn set_bit(&mut self, bit: u32, value: bool) {
+        self.value =
+            (((self.value as u32) & !(1 << (bit - 1))) | ((value as u32) << (bit - 1))) as f32;
+    }
+
+    pub fn get_bit(&self, bit: u32) -> bool {
+        ((self.value as u32 >> (bit - 1)) & 1) != 0
+    }
+}
```

Comment:
**thought**: I think we can store the value as `u32` instead of `f32`? I see a lot of conversion between these two numeric types, but am not really seeing a reason for doing so.